### ISSUE TITLE
Change cypress integration docs to use `@interactors/html` directly.

### ISSUE THREAD

**Issue by cowboyd** [thefrontside/interactors#97]
Currently, in our guide for cypress, we recommend importing interactors directly from `@interactors/cypress`

https://frontside.com/interactors/docs/integrations#cypress

While this works for `@interactors/html` it will not work as a setup for other interactor libraries such as `@interactors/material-ui` or a homegrown interactor library.

Let's instead instruct them to just add the following line to their [support file](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file)

```js
import '@interactors/with-cypress';
```

And then in their spec, instead of importing interactors from `@interactors/with-cypress`, they can just import them directly from `@interactors/html`. The advantage of this is that it is universal no matter what interactor library they use.

In other words, if I change from using `@interactors/html` to `@interactors/material-ui`, then the only thing I need to change is

```diff
--- import { Button } from '@interactors/html';
+++ import { Button } from '@interactors/material-ui';
```

inside my spec files.
